src/infer_response.h

```diff
@@ -338,10 +356,22 @@ class InferenceResponse {
 
   bool null_response_;
 
+  // A static variable counting the total number of responses created for the
+  // request.
+  static uint64_t total_responses_count;
+
 #ifdef TRITON_ENABLE_TRACING
   // Inference trace associated with this response.
   std::shared_ptr<InferenceTraceProxy> trace_;
 #endif  // TRITON_ENABLE_TRACING
+
+  // Representing the response index.
+  uint64_t response_idx_;
+
+  // Representing the request id that the response was created from.
+  uint64_t request_id_;
+
+  uint64_t response_start_;
```

Comment:
For sense of completion, add a descriptive comment here.

Reply:
Updated.